core/src/main/scala/org/apache/spark/executor/ExecutorExitCode.scala

```diff
@@ -47,8 +47,8 @@ object ExecutorExitCode {
 
   def explainExitCode(exitCode: Int): String = {
     exitCode match {
-      case UNCAUGHT_EXCEPTION => "Uncaught exception"
-      case UNCAUGHT_EXCEPTION_TWICE => "Uncaught exception, and logging the exception failed"
+      case UNCAUGHT_EXCEPTION => "Uncaught exception."
+      case UNCAUGHT_EXCEPTION_TWICE => "Uncaught exception, and logging the exception failed."
```

Comment:
Still trivial but these are not sentences. Here and below there is no reason to end with a period.

Reply:
Emm... So you mean we should remove all tailing dots in this function, and do appending on the caller side if necessary?

Reply:
I don't think they're needed at all. Separate stuff with spaces. But it doesn't matter

Reply:
Okay, let me revert changes in this function, just keeping it as-is.